dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Common/HelpLinkUrlFactory.cs

```diff
@@ -23,11 +23,11 @@ public static string Create(string diagnosticId)
             switch (diagnosticId)
             {
                 case DiagnosticId.FieldWithUnderscore:
-                    return $"{helpLinkUrlPrefix}variable-declarations-should-not-use-a-magic-number";
+                    return $"{helpLinkUrlPrefix}private-fields-should-be-prefixed-with-an-underscore";
```

Comment:
Fixed these incorrect URLs on the way past

Reply:
Nice one 👍